java-frontend/src/test/files/se/OptionalGetBeforeIsPresentCheck.java

```diff
@@ -82,6 +86,30 @@ private void usingOfNullableWithTest(@Nullable Object o) {
     }
   }
 
+  private void usingIsEmpty() {
+    Optional<Object> op = Optional.emtpy();
```

Comment:
typo in the name of the invoked method, should be `Optional.empty()`

Reply:
Thank you. Just fixed.